src/battle.py

```diff
@@ -139,6 +219,10 @@ def handle_melee(self, d, best_material, best_enemy_material, stats, enemy_stats
             if random.randint(0, self.target.discipline) == 0:
                 self.target.fatigue += 1
 
+            if is_heavy:
+                print self.target.name + "'s heavy armor is tiring the unit"
```

Comment:
Seems like a debugging print, please comment this out (or let me know if it's for some other purpose).